src/labelle/lib/devices/dymo_labeler.py

```diff
@@ -254,14 +254,15 @@ def __init__(
 
         if self._device is not None:
             # Retrieve device config
-            self._deviceConfig = get_device_config_by_id(self._device.id_product)
+            foundConfig = get_device_config_by_id(self._device.id_product)
+            if foundConfig is not None:
+                self._deviceConfig = foundConfig
+            else:
+                raise ValueError(f"Unsupported device type {self._device.id_product}")
```

Comment:
As before, style comment. Please use the following convention:
```
            if foundConfig is None:
                raise ValueError(f"Unsupported device type {self._device.id_product}")
            self._deviceConfig = foundConfig

```

Reply:
Will fix

Reply:
Fixed